frontend/orders-pane.js

```diff
@@ -15,11 +15,11 @@ class OrdersPane extends HTMLElement {
 		let shadow = this.attachShadow({mode: "open"});
 		let html = `
 			<div id="tabs">
-				<div id="tab_units" class="tab_units">units</div>
-				<div id="tab_plots" class="tab_plots">plots</div>
-				<div id="tab_nations" class="tab_nations">nations</div>
-				<div id="tab_economy" class="tab_economy">economy</div>
-				<div id="tab_game" class="tab_game">game</div>
+				<div id="tab_units" class="tab_units">Units</div>
```

Comment:
Quick spot-check looks like this brings capitalization consistency to the OrdersPane class, is this the convention for every HTMLElement?

Reply:
Originally I had the idea that capitalization is just an element of style and ::first-letter would be fine. However, ::first-letter breaks with the ::before, so the easiest way out was to eliminate that and just capitalize the strings in the HTML.

Doing a similar thing for the other classes is probably wise as well, although in some cases either capitalizing in JS or CSS is unavoidable since the strings in question have to align with the backend (are either sent from the backend or sent to the backend).